react/tic-tac-toe/src/app/screens/Login/components/CustomInput/styles.module.scss

```diff
@@ -0,0 +1,20 @@
+@import '../../../../../scss/colors';
+
+.inputContainer {
+  margin: auto;
+  padding: 10px 0;
+  width: 350px;
```

Comment:
No need to change this, but aware of this. If it is higher than 250px

max-width: 350px
width: 100%